hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -346,10 +350,16 @@ public SingleTransactionRecordBuilderImpl contractCreateResult(
     @NonNull
     public SingleTransactionRecordBuilderImpl transferList(@NonNull final TransferList transferList) {
         requireNonNull(transferList, "transferList must not be null");
-        transactionRecordBuilder.transferList(transferList);
```

Comment:
If this method no longer sets the transfer list, somewhere else the transfer list needs to be set on the `transactionRecordBuilder` (e.g. in the `build()` method)

Reply:
Right, it is done in line 189.

Reply:
Somehow I missed that 🤦 👍